### DISCUSSION TITLE
Logisim v3.7.0 is out

### DISCUSSION THREAD

**Discussion by BFH-ktt1** [logisim-evolution/logisim-evolution#1201] [Announcements]
Although V3.6.1 has been released recently we released v3.7.0 on the 12-10-2021. This release contains some bug fixes (amongst others in the polish version) and some nifty added features.

**Comment by aalcaineo** [comment]
Thanks!! great job!!

However, .dmg installer not working on mac though :S (same in LSE 3.7.1)

**Reply by davidhutchens** [reply]
@aalcaineo: Could you please provide more information?
What do you mean by "though :S"?
What version of macOS are you using?
Did you follow the instructions in the [README download section](https://github.com/logisim-evolution/logisim-evolution#download)?
Where does it fail and what are you seeing?
What version of macOS are you using?

Thank you reporting the problem and any help you can give us to reproduce it.

**Reply by ahanek** [reply]
same here, it is saying it is corrupted upon opening the app. Even .jar isn't working also. The reason I wanted to upgrade is because 
When going to the analyze and add inputs and outputs, minimize tab isn't showing. I thought this bug has been resolved with version 3.7.1

photo if the message is below:
![image](https://user-images.githubusercontent.com/35896978/138740210-d78d20c8-52d0-4a19-a84b-14db063d1daa.png)

**Reply by aalcaineo** [reply]
Exactly what @ahanek says.

I'm running MacOS BigSur 11.6 on a (Intel) Macbook Air I do not have problems installing 3.6 or 3.5 versions, so I expect to install them correctly as the dmg is obtained from the one you upload here.

Moreover, version 3.6 (both windows and mac) do not have the help (it promt me "help not found")

Hope this helps. 

bests!

**Reply by davidhutchens** [reply]
@aalcaineo and @ahanek: I have opened issue #1267 related to the macOS problem. Please continue this topic there.

@ahanek: Please check that you have at least Java version 16 for running the .jar.
